SConstruct

```diff
@@ -56,7 +61,9 @@ if os.name == 'posix' and env['platform'] == 'osx':
 globs = {
     'x11': '*.so.[0-9]*',
     'win64': '../bin/*-[0-9]*.dll',
-    'osx': '*.[0-9]*.dylib'
+    'osx': '*.[0-9]*.dylib',
+	 'x11_32': '*.so.[0-9]*',
```

Comment:
Mixed tabs and spaces or just the wrong number of spaces?

Should match indentation either way

Reply:
Wrong number of spaces that my code editor didn't show me while coding SConstruct file for 32bit platforms compilation